### DISCUSSION TITLE
按文档的theme配置遇到点问题？

### DISCUSSION THREAD

**Discussion by DengQingNian** [fect-org/fect#259] [Q&A]
抄的这个[页面](https://www.fect-org.com/en-us/guide/theme)的代码

![image](https://user-images.githubusercontent.com/117324885/203003761-86d9cbcb-fd6b-4f4e-a6ff-5dfc07ffcac0.png)

版本是 @fect-ui/vue@1.6.1

**Comment by nonzzz** [comment]
@RustyNailPlease  你这是使用了`vite-plugin-style-import` 插件吗。

**Reply by DengQingNian** [reply]
![image](https://user-images.githubusercontent.com/117324885/203005041-5d56aa0d-9720-4ccd-8866-aa5b0c326d9c.png)
我看了下是有的，这个是 那个nuxt的demo里的，会有什么影响吗

**Reply by nonzzz** [reply]
@RustyNailPlease  很抱歉 example太久没更新了。我会更新一个使用示例。因为useTheme属于composable。fect是没有包含这个style目录的。

**Reply by nonzzz** [reply]
@RustyNailPlease 可以查看[vite-plugin-style-import](https://github.com/vbenjs/vite-plugin-style-import) 文档。添加一个`ensureStyleFile = true`

**Reply by DengQingNian** [reply]
ok, 那先不用这个功能
